Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -658,8 +658,11 @@ void AODProducerWorkflowDPL::fillMCParticlesTable(o2::steer::MCKinematicsReader&
       }
       int statusCode = 0;
       uint8_t flags = 0;
-      if (!mcParticles[particle].isTransported()) {
+      if (!mcParticles[particle].isPrimary()) {
         flags |= 1 << 0; // mark as transported
```

Comment:
the comment is misleading. I would say `// mark as produced by transport`

Reply:
Yes, and also use o2::aod::mcparticle::enums::ProducedByTransport instead of `1 << 0`

Reply:
Thank you for comments. I have cleaned up the code